docs/source/api/v2/cdns_name_configs_monitoring.rst

```diff
@@ -80,29 +83,36 @@ Response Structure
+
 :trafficMonitors: An array of objects representing each Traffic Monitor that monitors this CDN (this is used by Traffic Monitor's "peer polling" function)
 
-	:fqdn:     An :abbr:`FQDN (Fully Qualified Domain Name)` that resolves to the IPv4 (and/or IPv6) address of the server running this Traffic Monitor instance
-	:hostname: The hostname of the server running this Traffic Monitor instance
-	:ip6:      The IPv6 address of this Traffic Monitor - when applicable
-	:ip:       The IPv4 address of this Traffic Monitor
-	:port:     The port on which this Traffic Monitor listens for incoming connections
-	:profile:  A string that is the :ref:`profile-name` of the :term:`Profile` assigned to this Traffic Monitor
-	:status:   The status of the server running this Traffic Monitor instance
+	:cachegroup: The :term:`Cache Group` to which this Traffic Monitor belongs
+	:fqdn:       An :abbr:`FQDN (Fully Qualified Domain Name)` that resolves to the IPv4 (and/or IPv6) address of the server running this Traffic Monitor instance
+	:hostname:   The hostname of the server running this Traffic Monitor instance
+	:ip6:        The IPv6 address of this Traffic Monitor - when applicable
+	:ip:         The IPv4 address of this Traffic Monitor
+	:port:       The port on which this Traffic Monitor listens for incoming connections
+	:profile:    A string that is the :ref:`profile-name` of the :term:`Profile` assigned to this Traffic Monitor
+	:status:     The status of the server running this Traffic Monitor instance
 
 :trafficServers: An array of objects that represent the :term:`cache servers` being monitored within this CDN
 
-	:cacheGroup:    The :term:`Cache Group` to which this :term:`cache server` belongs
-	:fqdn:          An :abbr:`FQDN (Fully Qualified Domain Name)` that resolves to the :term:`cache server`'s IPv4 (or IPv6) address
-	:hashId:        The (short) hostname for the :term:`cache server` - named "hashId" for legacy reasons
-	:hostName:      The (short) hostname of the :term:`cache server`
-	:interfacename: The name of the network interface device being used by the :term:`cache server`'s HTTP proxy
-	:ip6:           The :term:`cache server`'s IPv6 address - when applicable
-	:ip:            The :term:`cache server`'s IPv4 address
-	:port:          The port on which the :term:`cache server` listens for incoming connections
-	:profile:       A string that is the :ref:`profile-name` of the :term:`Profile` assigned to this :term:`cache server`
-	:status:        The status of the :term:`cache server`
-	:type:          A string that names the :term:`Type` of the :term:`cache server` - should (ideally) be either ``EDGE`` or ``MID``
+	:cacheGroup:       The :term:`Cache Group` to which this :term:`cache server` belongs
+	:deliveryServices: An array of objects which contain the XML IDs of the delivery services to which this cache server is assigned
+
+		:xmlId: A string which is the XML ID of the delivery service
```

Comment:
Our API guidelines say

> If a property is missing from an object representation it indicates the absence of that property, and because of that there must be a very convincing argument if and when that is the case.

So being that it's unusual for properties to be undefined in our API, would you mind making note of that? People might reasonably expect `[]` - or even `null` - instead, since that's recommended. Omitting is consistent with the rest of that endpoint, unfortunately, though.

Reply:
Done